libs/cart/src/facades/cart/cart.facade.ts

```diff
@@ -65,33 +40,32 @@ export class DaffCartFacade implements DaffStoreFacade<Action> {
   availablePaymentMethods$: Observable<DaffCart['available_payment_methods']>;
   isCartEmpty$: Observable<boolean>;
 
-  constructor(private store: Store<DaffCartReducersState>) {
-    this.loading$ = this.store.pipe(select(selectCartLoading));
-    this.cart$ = this.store.pipe(select(selectCartValue));
+  constructor(private store: Store<DaffCartReducersState<T>>) {
+    this.loading$ = this.store.pipe(select(getDaffCartSelectors<T>().selectCartLoading));
```

Comment:
Can the initial call be moved to a const either in the ngOnInit or constructor?

Reply:
done